### ISSUE TITLE
Feature request: Support for Windows and macOS

### ISSUE THREAD

**Issue by b-long** [Douane/Douane#68]
This project looks incredible, great job!  

My sense is that this effort could be a highly sought after alternative to proprietary offerings (e.g. [Little Snitch](https://www.obdev.at/products/littlesnitch/index.html)).  It would be fantastic if you could support the other major platforms and may motivate additional developers to contribute.

Will you consider supporting other platforms and perhaps encourage such development?

**Comment by zedtux** [comment]
Thank you @b-long for this issue 👍 

Initially I wrote Douane because Linux is lacking a good application firewall and because I'm using Little Snitch on macOS (and for Windows, I don't know if any kind of app like this exists).

For sure you have a point here, but Douane is not mature enough for this today.
I have implemented the only the Minimum Valuable Product and compared to Little Snitch (using on Mac decided me to build Douane for Linux) it will miss a lot of features.

Unfortunately for now I don't have time to continue the development of Douane, but the project is not abandoned.
I'll always take time for contributions (as I already did) and I'll come back on this project later.

That said, I would love to see Douane everywhere! Have an open source and free app like this is **must** as our privacy is part of the high priority for all of us and it **must** be free in all the senses of the word.

Personally I will not port it on Windows for sure as I'm not using it and I never developed anything on Windows.
I could make it for macOS (the day I'm coming back on developing Douane) but it would happen only after having done the major point of [the Douane roadmap](https://github.com/Douane/Douane/wiki/Roadmap).

In the case we would go this way, we will require to have a test suite for all the OS in order to be sure that change this, will not break that on this OS.

Let's keep this issue open, and see.

**Comment by rjzak** [comment]
Not pertaining to Douane, but there are two similar apps to Douane & Little Snitch for Windows: Windows Firewall Control https://www.binisoft.org/wfc.php and GlassWire https://www.glasswire.com/. Neither are as thorough or detailed as Little Snitch.

**Comment by jpaulo10** [comment]
I use tripmode in windows/mac to limit wich applications can access my network. Certainly it doesn't have as many features as Little Snitch, but it solves my problems.

**Comment by b-long** [comment]
Thanks @jpaulo10 , I appreciate you highlighting them.  Another effort, similar to Douane is OpenSnitch: https://github.com/evilsocket/opensnitch .  There may be bits of overlap between the two that could be shared.

**Comment by zedtux** [comment]
@b-long what about a PR updating the `README.md` file pointing to a Wiki page with this list of options?

**Comment by zedtux** [comment]
This repo has moved to GitLab and this issue has been imported. You can access it [here](https://gitlab.com/douaneapp/Douane/issues/68).
